docs/electrical/microcontrollers/microcontroller-architecture.md

```diff
@@ -12,7 +12,18 @@ The STM32 family of microcontrollers has been identified as a
 strong candidate for this application due to it being very well
 supported, its wide selections, and its low cost.
 
-## MCU Candidates
+## MCU Choice
+### [STM32L051C6T6](https://estore.st.com/en/stm32l051c6t6-cpn.html)
+- ARM Cortex M0+
+- 32KB Flash Memory
+- 32 MHz
+- Up to 51 GPIO pins
+- 88 micro Amp/MHz in run mode
```

Comment:
HTML has support for &#956;A, so it's probably better to use that (fixed in [#5](https://github.com/ncsurobotics/SW9-Documentation/pull/5), don't worry about it here)